molgenis_documentation/technical_documentation.md

```diff
@@ -22,88 +22,36 @@ The “hairball” is the twisted, tangled maze of teaching that is going to tak
 * Contribution policies (if your project accepts outside contributions).
 
 
-# MOLGENIS Technical documentation
+# MOLGENIS user documentation
 ---
+## Introduction
+**
+This document is a detailed description of the web-based MOLGENIS software. Here you will learn, step-by-step, what MOLGENIS is, and the many options it offers you to explore and manipulate your data.
+**
 
-## Introduction (First contact)
-For years the amount of biological data has been increasing.
+### What is MOLGENIS?
+Molecular Genetics Information Systems, or MOLGENIS for short, is a web-based software toolkit designed to provide biologists with user friendly and scalable software infrastructures to capture, exchange, and exploit the large amounts of data that is being produced by scientific organizations all around the world. 
 
-### User example
-A biologist has generated gigabytes of data, and have processing is now searching for a way to use it in a clear and comprehensive way…. he wants to use MOLGENIS of course.
+### Why MOLGENIS?
+Why should you use MOLGENIS? One of the key features is that it uses an extensible model system, allowing users to model their data however they want. This creates flexibility that other, more static, database applications often lack. Its web-based, meaning you setup a server, install and configure MOLGENIS, load your data and share it with the world. If your data is ready, setting up a useful online research database can be done in a matter of days. Besides storing your data, MOLGENIS also allows for the creation of R and Python scripts that interact with your data. This enables you to run statistical analysis, or create plots based on your data within the online environment.
 
-### What can MOLGENIS be used for
-MOLGENIS has a plethora of use cases, several of these are described below
+MOLGENIS takes away the hassle of storing and analysing data by taking care of the storing and querying part. Allowing researchers to focus on the data itself.
 
-#### how to create a biobank catalogue
-Creating a biobank catalogue text + screenshots + example projects
+### Should I use MOLGENIS?
+If you are a biologist, a bioinformatician, a researcher, or anyone else who has a lot of biological data on their hands, then MOLGENIS is a software package that will help you in setting up an online research database in no time at all, making your data query-able and allowing you to share your data with collaborators effortlessly. By mastering the MOLGENIS software toolkit you will be able to store, edit, analyse, and share your data faster then ever before. If one of the following use cases applies to you, then yes it is worth the effort to learn MOLGENIS.
+
+#### Biobank catalogue
+Biobank catalogue text + screenshots + example projects
```

Comment:
These parts should cover the question people can have about wether MOLGENIS is something for them. By showing examples of biobanks and other research portals I want to give them something to relate to.